src/Containers/ForecastPage/index.tsx

```diff
@@ -0,0 +1,21 @@
+import WeatherCard from "../../Components/WeatherCard";
+import Header from "../../Components/Header";
+import Location from "../../Components/Location";
+
+const ForecastPage = () => {
+  return (
+    <div>
+      <Header />
+      <Location />
+      <div className="weather">
+        <WeatherCard />
+        <WeatherCard />
+        <WeatherCard />
+        <WeatherCard />
+        <WeatherCard />
```

Comment:
Why not use loop here for cards? even when you get the data you'll be using that?

Reply:
Just for now i have use it like this but i will remind it  from the next time.